src/table/toolbar/table-toolbar.component.ts

```diff
@@ -64,6 +64,8 @@ import { I18n, Overridable } from "../../i18n/i18n.module";
 export class TableToolbar {
 	@Input() model: TableModel;
 
+	@Input() batchText = "items selected";
```

Comment:
We'll want to add this to the translations as well, since it'll likely be a single translation unit for most cases